src/js/menu/menu-item.js

```diff
@@ -17,7 +17,18 @@ class MenuItem extends Button {
 
   constructor(player, options) {
     super(player, options);
+
+    this.selectable = options['selectable'];
+
     this.selected(options['selected']);
+
+    if (this.selectable) {
+      // TODO: May need to be either menuitemcheckbox or menuitemradio,
+      //       and may need logical grouping of menu items.
+      this.el_.setAttribute('role', 'menuitemcheckbox');
```

Comment:
locally, I've played around with the `menu` itself to a role of `listbox` and these to a role of `option`.
Using ChromeVox for testing it actually produced decent results since it told you how many options and whether they're selected or not.